pkg/network/server.go

```diff
@@ -295,6 +308,14 @@ func (s *Server) Start() {
 	go s.relayBlocksLoop()
 	go s.bQueue.Run()
 	go s.bSyncQueue.Run()
+	go s.bFetcherQueue.Run()
+	if s.ServerConfig.NeoFSBlockFetcherCfg.Enabled && s.blockFetcher != nil {
```

Comment:
Why do you need the second check? Is it possible that `s.ServerConfig.NeoFSBlockFetcherCfg.Enabled` and `s.blockFetcher` is nil?